quiz.css

```diff
@@ -0,0 +1,68 @@
+body {
+  font-family: sans-serif;
+}
+
+article {
+  width:30%;
+  border: 1px solid lightblue;
+  min-width: 250px;
+  display: inline-block;
+  margin-bottom: 5px;
+}
+
+img {
+  display: block;
+  margin: auto;
+  height: 200px;
+}
+
+.cardtitle {
+  border: 1px solid lightgray;
+  text-align: center;
+  margin: 0;
+  padding: 15px;
+}
+
+.availability {
+  color: gold;
+  background-color: grey;
+  padding: 5px 3px;
+}
+
+.productDescription {
+  text-align: justify;
+  padding: 4px;
+  line-height: 1em;
+}
+
+.indent {
+  padding-left: 10px;
+  margin-top: -10px;
+}
+
+.specs {
+    padding-left: 10px;
+    margin-top: -10px;
```

Comment:
Remember to keep consistent spacing and indentation. There was also a <p> tag in the main file that wasn't lined up right. Just something to keep an eye on as you move forward!